src/NUnitFramework/nunitlite/TextUI.cs

```diff
@@ -489,6 +489,11 @@ private void DisplayTestResult(ITestResult result)
             string reportID = (++_reportIndex).ToString();
             int numAsserts = result.AssertionResults.Count;
 
+#if PORTABLE && !NETSTANDARD1_6
+            ColorStyle style = GetColorStyle(resultState);
```

Comment:
Ah, I see the answer to my question. This is because we don't have the proper callstack in PORTABLE.